Networking/Networking/Model/ShippingLabel/Enums/ShippingLabelRefundStatus.swift

```diff
@@ -4,6 +4,7 @@ import Codegen
 /// The status of shipping label refund.
 public enum ShippingLabelRefundStatus: GeneratedFakeable {
     case pending
+    case unknown
```

Comment:
Could you explain why we have to add the `unknown` case here? Thank you!

Based on my understanding, the `ShippingLabelRefund.status` value from the core data layer is a non-optional String with a default value (empty string). When this `ShippingLabelRefund` instance is deleted, and the context is saved, but the object is still accessible in memory, the value for `status` will be an empty string. 

In the above case, the app will crash due to the usage of `assertionFailure`. But this will happen only in debug mode and won't affect release builds. 

I am not sure where we use this `unknown` case.

Reply:
From my testing the `assertionFailure` also caused crashes in the unit tests so I could not proceed and had to add this solution. IMO it makes more sense to add an unknown case and log the error so that we can keep track of it, rather than let it fail silently. WDYT?

Reply:
> From my testing the assertionFailure also caused crashes in the unit tests so I could not proceed and had to add this solution.

Good point. I didn't realise this. Thanks!

> IMO it makes more sense to add an unknown case and log the error so that we can keep track of it, rather than let it fail silently. WDYT?

Sounds good to me.